### DISCUSSION TITLE
Customizing the payment popup

### DISCUSSION THREAD

**Discussion by amonam** [Glench/ExtPay#130] [Ideas]
Please provide additional options for customizing the payment popup, specifically for modifying the text. For instance, an option for 'Charged immediately - cancel anytime' would be helpful.
We have found that we are losing a significant number of customers because they do not connect well with our current user interface, and they find it difficult to trust putting their card information on it.

Here is an example I found on Pinterest that I wanted to share for inspiration.
![image](https://user-images.githubusercontent.com/19654690/236502947-b649affe-f626-4cd2-8449-f09dc3ab77f1.png)

**Comment by burkybang** [comment]
You can stylize that page however you want using your extension. You could even change wording using JavaScript.

**Reply by takkbbl** [reply]
That sounds promising. Can you give us a hint how to do it?

**Reply by burkybang** [reply]
https://stackoverflow.com/questions/7619095/how-to-inject-css-into-webpage-through-chrome-extension

https://developer.chrome.com/docs/extensions/mv3/content_scripts

**Reply by takkbbl** [reply]
thank you
